feature-gate/Dockerfile.tilt

```diff
@@ -1,11 +1,11 @@
 # Create minimal deployment
 # with just the service binary included
 
-FROM debian:buster-slim
+FROM ubuntu:hirsute
```

Comment:
This is because compiling stuff on my workstation (Ubuntu 21.04) causes the binary to get linked to too new of a glibc version for debian:buster (or even debian:bullseye).